ffmpeg fails linking vapoursynth: undefined reference to __imp_vsscript_* functions

**ffmpeg-git/build-static-64bit/ab-suite.make.log**
```
{...}
LD	ffmpeg_g.exe
LD	ffplay_g.exe
LD	ffprobe_g.exe
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x2f): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x35): undefined reference to `__imp_vsscript_finalize'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x56): undefined reference to `__imp_vsscript_init'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x7e): undefined reference to `__imp_vsscript_getApiVersion'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xb4): undefined reference to `__imp_vsscript_getVSApi2'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xf1): undefined reference to `__imp_vsscript_evaluateFile'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x121): undefined reference to `__imp_vsscript_getOutput'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x2f): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x552): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x35): undefined reference to `__imp_vsscript_finalize'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x55f): undefined reference to `__imp_vsscript_finalize'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x56): undefined reference to `__imp_vsscript_init'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x7e): undefined reference to `__imp_vsscript_getApiVersion'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xb4): undefined reference to `__imp_vsscript_getVSApi2'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xf1): undefined reference to `__imp_vsscript_evaluateFile'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x121): undefined reference to `__imp_vsscript_getOutput'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x552): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x55f): undefined reference to `__imp_vsscript_finalize'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x2f): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x35): undefined reference to `__imp_vsscript_finalize'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x56): undefined reference to `__imp_vsscript_init'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x7e): undefined reference to `__imp_vsscript_getApiVersion'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xb4): undefined reference to `__imp_vsscript_getVSApi2'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xf1): undefined reference to `__imp_vsscript_evaluateFile'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x121): undefined reference to `__imp_vsscript_getOutput'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x552): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x55f): undefined reference to `__imp_vsscript_finalize'
collect2.exe: error: ld returned 1 exit status
collect2.exe: error: ld returned 1 exit status
make: *** [/build/ffmpeg-git/Makefile:141: ffprobe_g.exe] Error 1
make: *** Waiting for unfinished jobs....
make: *** [/build/ffmpeg-git/Makefile:141: ffplay_g.exe] Error 1
collect2.exe: error: ld returned 1 exit status
make: *** [/build/ffmpeg-git/Makefile:141: ffmpeg_g.exe] Error 1
LD	ffmpeg_g.exe
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x2f): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x35): undefined reference to `__imp_vsscript_finalize'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x56): undefined reference to `__imp_vsscript_init'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x7e): undefined reference to `__imp_vsscript_getApiVersion'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xb4): undefined reference to `__imp_vsscript_getVSApi2'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0xf1): undefined reference to `__imp_vsscript_evaluateFile'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x121): undefined reference to `__imp_vsscript_getOutput'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x552): undefined reference to `__imp_vsscript_freeScript'
G:/MABS/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/14.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: libavformat/libavformat.a(vapoursynth_alt.o):vapoursynth_al:(.text.unlikely+0x55f): undefined reference to `__imp_vsscript_finalize'
collect2.exe: error: ld returned 1 exit status
make: *** [/build/ffmpeg-git/Makefile:141: ffmpeg_g.exe] Error 1
```

[logs.zip](https://github.com/user-attachments/files/16824998/logs.zip)

--- a/build/media-suite_compile.sh
+++ b/build/media-suite_compile.sh
@@ -2254,8 +2254,6 @@ if [[ $ffmpeg != no ]]; then
         enabled libsvtav1 || do_removeOption FFMPEG_OPTS_SHARED "--enable-libsvtav1"
         enabled libsvtvp9 || do_removeOption FFMPEG_OPTS_SHARED "--enable-libsvtvp9"
 
-        enabled vapoursynth && do_patch "https://raw.githubusercontent.com/m-ab-s/mabs-patches/master/ffmpeg/0001-Add-Alternative-VapourSynth-demuxer.patch" am
-
         if enabled openal &&
             pc_exists "openal"; then
             OPENAL_LIBS=$($PKG_CONFIG --libs openal)

